third_party/nvidia/language/cuda/gdc.py

```diff
@@ -0,0 +1,13 @@
+from triton.language import core
+
+
+@core.extern
+def gdc_wait(_builder=None):
+    return core.inline_asm_elementwise("griddepcontrol.wait; // dummy $0", "=r", [], dtype=core.int32, is_pure=False,
```

Comment:
why do you need to return the dummy value?

Reply:
The `inline_asm_elementwise` requires a return value, so I return a dummy value.

Reply:
I'm not sure if the limitation is in the core op or there is an assumption in the compiler for the inline_asm_elementwise ops.

Reply:
I just changed this to not return thanks for catching it.